tools/gitpod/gitpod.Dockerfile

```diff
@@ -39,6 +39,9 @@ RUN conda activate ${CONDA_ENV} && \
     python setup.py build_ext --inplace && \
     ccache -s
 
+# Install dependencies for Live-Preview on Gipod
+RUN python -m pip install esbonio   
+
```

Comment:
I'd just as soon leave this out. `esbonio` is an editor dependency so I'm inclined to let the editor handle it's dependencies itself. AFAICT this is fine because `esbonio` is being installed currently in the gitpod env at doc build time without any intervention on our part.

Reply:
Not able to get your views clearly ross. but I am hoping you are suggesting to remove it. Seeing "let editor handle its dependencies". Which I abide by.

As stated above even I wasn't sure of this and needed some opinions. 

Removing it for now

Reply:
Sorry for being unclear - yes, let's remove it!